doc/srv6/SRv6_uSID.md

```diff
@@ -0,0 +1,112 @@
+# SONiC uSID
+
+## Table of Content
+- [Overview](#Overview)
+- [Scope](#Scope)
+- [Design](#Design)
+- [Example](#Example)
+
+## Revision
+
+| Rev  |   Date    |              Author              | Change Description      |
+| :--: | :-------: | :------------------------------: | :---------------------: |
+| 0.1  | 7/17/2022 | Shitanshu Shah, Reshma Sudarshan |  Initial version        |
+
+
+## Overview
+SRv6 uSID (micro-segment) is extension of the SRv6 network programming model [SRv6 uSID instructions IETF draft](https://datatracker.ietf.org/doc/draft-filsfils-spring-net-pgm-extension-srv6-usid/). uSID is a compressed SID value which can be for example carried in 16-bits (unlike full IPv6 address to represent a SID). uSID as is designed scales well with much lower MTU overhead required per uSID carrier. uSID carrier is 128-bit IPv6 address that can carry upto 6 uSIDs [Refer to Example for more details]

```

Comment:
Looks like https://datatracker.ietf.org/doc/draft-filsfils-spring-net-pgm-extension-srv6-usid is not an IETF adopted draft yet and it is currently individual draft under IETF spring WG, why would we need this in SONiC until it becomes IETF adopted draft?

Reply:
Indeed, the HLD should also point to the IETF draft [draft-ietf-spring-srv6-srh-compression-02](https://datatracker.ietf.org/doc/html/draft-ietf-spring-srv6-srh-compression-02), which is IETF Working Group draft and standardizes it.

Reply:
Venkatesesan, also it might be interesting for you to listen to this presentation from MPLS World Congress if you have the time. 
[Alibaba: Full stack SRv6 towards a 'Predictable Network' - YouTube](https://www.youtube.com/watch?v=izE4eh1q6ts)
Alibaba explains their plans to deploy SRv6 uSID, not only with commercial routers, but also with a SONiC based platform. This deployment is happening in early CY23, and hence throughout the next few months the community will work hard on the different missing components (e.g., FRR).
Thanks!!

Reply:
updated reference to both IETF drafts. Please review/approve appropriately.
Thanks,